### ISSUE TITLE
Kaggle training mask is black

### ISSUE THREAD

**Issue by HaixiongLi** [milesial/Pytorch-UNet#118]
After training with the KITTI data set, I can get better results, but the mask obtained by kaggle training is black, and I try to output the full_mask, which is the image before comparing with the threshold of 0.5. When I see the image, it is even gray. My epochs = 5。
At the same time, it is worth noting that after 500, the loss will basically not decrease, and will remain between 0.4-0.5.
Why?  Looking forward to your reply

**Comment by milesial** [comment]
Maybe your data pipeline is wrong. Are you able to overfit a single example image / mask ?
